kubeval/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.10
+
+ENV KUBEVAL_VERSION=0.14.0 \
+    KUBECTL_VERSION=1.16.2
+
+RUN apk add --no-cache \
```

Comment:
I'd probably inline this, as it seems more compact to me. But I'll leave it up to you 🙂

Reply:
It seems we don't need to install any additional packages in the end.

Reply:
Even better 😄